frontend/public/components/modals/index.ts

```diff
@@ -55,6 +55,11 @@ export const podSelectorModal = (props) =>
     m.podSelectorModal(props),
   );
 
+export const rollBackRCModal = (props) =>
```

Comment:
nit: I prefer `rollbackRCModal` if "Rollback" is one word. That also matches the case of the API resource `DeploymentConfigRollback`. (Same for some of the other names below.)